### PR TITLE
Access DOM Element via ElementRef

### PR DESCRIPTION
**Pull request by alQlagin**

### CODE REVIEW

File: snippets/access-dom-element.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Access DOM Element
+author: Alex Kulagin
```
Comment (thekiba) [inline snippets/access-dom-element.md]:
```suggestion
author: alQlagin
```

**Review by thekiba — CHANGES_REQUESTED** [review]

File: snippets/access-dom-element.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private elementRef: ElementRef<HTMLElement>,
+  ) { }
+}
+```
+Be aware! Direct dom manipulations are bad practice
+
+# ComponentCode
+```typescript
+import { Directive, ElementRef, OnInit, OnDestroy } from '@angular/core';
+import Tooltip from 'tooltip.js';
```
Comment (thekiba) [inline snippets/access-dom-element.md]: It will works on demo?

Comment (alQlagin) [reply snippets/access-dom-element.md]: Here is working demo
https://stackblitz.com/edit/angular-tooltipjs-directive?file=src%2Fapp%2Ftooltip.directive.ts

**Review by kirjs — APPROVED** [review]
I dropped the demo because we can't require the library

**Review by kirjs — COMMENTED** [review]
TODO: Add a demo

**Comment by NothingEverHappens** [comment]
Please add a demo?

**Comment by kirjs** [comment]
add a demo?

**Comment by alQlagin** [comment]
Duplicate of #241

**Pull request closed without merging**